### DISCUSSION TITLE
Enable multiple selection of ApexLogs

### DISCUSSION THREAD

**Discussion by lz90** [forcedotcom/salesforcedx-vscode#5862] [Ideas]
Problem:
to download more than one ApexLog it is necessary to use many time the GetLogs command

Possible Solution:
to use the **canPickMany?: boolean** parameter in the forceApexLogGet.ts

**Comment by daphne-sfdc** [comment]
Hi @lz90, Thank you for the feature request and proposed solution.  We'll let you know when we implement this product improvement.

**Comment by git2gus** [comment]
This issue has been linked to a new work item: W-15069117
